queries.sql

```diff
@@ -7,4 +7,43 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agmon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg > 10.2 AND weight_kg < 17.5;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg > 10.2 AND weight_kg < 17.5;
+
+/***** TRANSACTIONS *****/
+
+BEGIN; 
+UPDATE animals
+SET species = 'unspecified'; 
+SELECT species FROM animals; 
+ROLLBACK;
+SELECT species FROM animals;
```

Comment:
- [ ] Great work implementing this change that was requested by the first reviewer. However, you forgot to include the screenshot of this transaction in your pull request. Kindly include the screenshot as it is one of the project's requirements. 👍 😎